webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/detail/AnnotationDetailEditorPanel.java

```diff
@@ -663,12 +666,6 @@ public void actionDelete(AjaxRequestTarget aTarget)
             }
         }
 
-        // If the deleted FS is a relation, we don't have to do anything. Nothing can point to a
```

Comment:
At least the comment should stay. The code/comment don't do anything, but they document that actually nothing *should* be done which is quite helpful.